packages/resolve-redux/package.json

```diff
@@ -22,37 +22,33 @@
   ],
   "homepage": "https://github.com/reimagined/resolve/tree/master/packages/resolve-redux#readme",
   "peerDependencies": {
-    "react": "^15.3.0 || ^16.0.0",
-    "react-dom": "^15.3.0 || ^16.0.0"
+    "react": ">=15",
+    "react-dom": ">=15"
   },
   "devDependencies": {
-    "babel-preset-es2015": "^6.24.1",
-    "babel-preset-react": "^6.24.1",
-    "babel-preset-stage-0": "^6.24.1",
-    "chai": "^3.5.0",
+    "chai": "^4.1.2",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
-    "jest": "^22.1.4",
+    "jest": "^22.4.2",
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
-    "react-test-renderer": "^15.6.2",
-    "redux": "^3.6.0",
-    "seamless-immutable": "^7.1.2",
-    "sinon": "^3.3.0",
-    "uuid": "^3.1.0"
+    "react-test-renderer": "^16.2.0",
+    "redux": "^3.7.2",
+    "seamless-immutable": "^7.1.3",
+    "sinon": "^4.4.2",
+    "uuid": "^3.2.1"
   },
   "dependencies": {
-    "apollo-cache-inmemory": "^1.1.0",
-    "apollo-client": "^2.0.2",
-    "apollo-link-http": "^1.1.0",
+    "apollo-cache-inmemory": "^1.1.9",
+    "apollo-client": "^2.2.5",
+    "apollo-link-http": "^1.5.2",
     "diff-json": "^0.1.11",
-    "graphql": "^0.11.7",
-    "graphql-tag": "^2.5.0",
+    "graphql": "^0.13.1",
```

Comment:
Is `graphql` still actual? I know most likely it had been remained wrongly after read-modules updating. Also with `apollo`.